### PR TITLE
Added more logging

### PR DESCRIPTION
**Pull request by kberryUSGS**

### CODE REVIEW

File: include/usgscsm/UsgsAstroSarSensorModel.h [hunk 1]
```diff
@@ -23,9 +25,9 @@ class UsgsAstroSarSensorModel : public csm::RasterGM, virtual public csm::Settab
 
     virtual std::string getModelState() const;
 
-    static std::string constructStateFromIsd(const std::string imageSupportData, csm::WarningList *list);
+    std::string constructStateFromIsd(const std::string imageSupportData, csm::WarningList *list);
```
Comment (kberryUSGS) [inline include/usgscsm/UsgsAstroSarSensorModel.h:28]: These were both not static in the Framer and LineScanner, and it removed the ability to test using m_logger. Thoughts?

File: tests/SarTests.cpp [hunk 2]
```diff
@@ -15,25 +15,28 @@
...
-  catch(...) {
-    for (auto &warn: warnings) {
-      std::cerr << "Warning in " << warn.getFunction() << std::endl;
-      std::cerr << "  " << warn.getMessage() << std::endl;
-    }
-    FAIL() << "constructStateFromIsd errored";
-  }
-  EXPECT_TRUE(warnings.empty());
-}
+// UsgsAstroSarSensorModel is the only sensor model with this function static. Static throws off the
```
Comment (kberryUSGS) [inline tests/SarTests.cpp:18]: See this comment. Thoughts?

File: src/UsgsAstroPlugin.cpp [hunk 3]
```diff
@@ -273,97 +293,111 @@ std::string UsgsAstroPlugin::convertISDToModelState(const csm::Isd &imageSupport
...
+  
+  MESSAGE_LOG("ISD String: {}", stringIsd);
+  if (modelName == UsgsAstroFrameSensorModel::_SENSOR_MODEL_NAME) {
+    UsgsAstroFrameSensorModel *model =  new UsgsAstroFrameSensorModel();
+    try {
+      MESSAGE_LOG("Trying to construct a UsgsAstroFrameSensorModel");
+      model->replaceModelState(model->constructStateFromIsd(stringIsd, warnings));
+      std::shared_ptr<spdlog::logger> logger = model->getLogger();
+      if (logger) {
+        logger->info("Constructed model: {}", modelName);
```
Comment (jessemapel) [inline src/UsgsAstroPlugin.cpp:307]: Why does this not use the logging macro?

File: tests/SarTests.cpp (hunk 2 above)
Comment (jessemapel) [reply tests/SarTests.cpp:18]: This doesn't need to be static

File: src/UsgsAstroPlugin.cpp (hunk 3 above)
Comment (kberryUSGS) [reply src/UsgsAstroPlugin.cpp:307]: Ah! This was the only part of the plugin that was previously logged before I added to the Macro. Good catch, thanks!

**Pull request merged**